tests/contrib/test_appengine.py

```diff
@@ -922,11 +923,10 @@ def test_decorator_from_client_secrets_bad_type(self):
             'oauth2client.contrib.appengine.clientsecrets.loadfile')
         with loadfile_patch as loadfile_mock:
             loadfile_mock.return_value = ('badtype', None)
-            self.assertRaises(
-                AppEngineInvalidClientSecretsError,
-                OAuth2DecoratorFromClientSecrets,
-                'doesntmatter.json',
-                scope=['foo_scope', 'bar_scope'])
+            with self.assertRaises(AppEngineInvalidClientSecretsError):
+                OAuth2DecoratorFromClientSecrets('doesntmatter.json',
```

Comment:
prefer to break after the opening `(` to avoid these scrunched-up hanging indents.